### ISSUE TITLE
Best way to include pure html post

### ISSUE THREAD

**Issue by sgfin** [aron-bordin/neo-hpstr-jekyll-theme#42]
Hello,

Apologies for the ignorant question, but I am a researcher interested in using this theme to host/log my work for easier sharing/reproducibility.  Much of my work is done in R Notebooks, which are designed to export to standalone single-file html pages for easy deployment.

My question is:  would it be possible to take these plain html pages and have the website treat them as "posts"?  As it is, I can toss the static file in a folder in the github repo and access/render it directly,( e.g. https://sgfin.github.io/files/Aetna_Epilepsy_Cohort_Building.nb.html), but this is obviously not the same as creating a post, or even having it as a page on the site with the normal header/footer/etc.

Thanks,
Sam

**Comment by nrbrd** [comment]
Hi,

I'm not able to test it right now, I'm really busy :/ But you may try to load the html inside an iframe (http://stackoverflow.com/q/30233461) or check if it's possible to export/save the notebook as a markdown (I know that it's possible in python)

**Comment by sgfin** [comment]
Thank you for taking the time to answer.  Exporting as markdown is a wee bit clunky, bc I have to manually update the image links (at least until I make an automation script or something) -- but it gets the job done! 

Thanks,
Sam

**Comment by sgfin** [comment]
OK, so I looked into this a tiny bit on the jekyll side, and saw that it is perfectly valid to add a html file to the post folder, provided that (a) the title (before ".html") is of the same format, and (b) you still include a YAML header just as with the .md files.

I tried this, and it worked, but the formatting looked super weird.  I'm guessing it's a simple fix for someone who knows what they're doing.

I know you're busy, but if/when you ever get a chance to look at this, I'd really appreciate it.  Happy to provide more info or an example file later.

Thanks,
Sam
